aws/load-balancer.tf

```diff
@@ -43,6 +43,23 @@ resource "aws_lb_listener" "rancher_https" {
   }
 }
 
+resource "aws_s3_bucket" "rancher_lb_access_logs" {
+  count         = "${var.rancher_lb_access_logs_bucket_create ? 1 : 0}"
+  bucket        = "${var.rancher_lb_access_logs_bucket}"
+  acl           = "private"
+  force_destroy = true
+
+  lifecycle_rule {
+    enabled = true
+
+    expiration {
+      days = 10
+    }
+  }
+
+  tags = "${merge(map("Name", "rancher"), var.cloud_tags)}"
```

Comment:
The bucket name should probably not be hardcoded. Using `var.rancher_lb_access_logs_bucket` should be fine.